pkg/perfscaleanalysis/perfscalehelpers.go

```diff
@@ -120,7 +120,7 @@ func DownloadPerfScaleData(storagePath string, reportEnd time.Time) error {
 	if err != nil {
 		return errors.Wrap(err, "error marshalling scale jobs")
 	}
-	err = ioutil.WriteFile(filepath.Join(storagePath, ScaleJobsFilename), file, 0600)
+	err = ioutil.WriteFile(filepath.Join(storagePath, ScaleJobsFilename), file, 0o600)
```

Comment:
Looks like this is because of https://github.com/golang/go/issues/33363

Such a weird syntax.  0 as a prefix has denominated octals for decades :-\